packages/@aws-cdk/region-info/lib/user-data.ts

```diff
@@ -0,0 +1,95 @@
+import fs = require('fs');
+import jsonschema = require('jsonschema');
+import { IRegionInfo } from './iregion-info';
+import builtIns = require('./static.generated');
+
+export interface IUserSuppliedRegionInfo {
```

Comment:
How come this is a different interface? Seems duplicated.

Reply:
Evvvvverything is optional here!